linked_list/linked_list.py

```diff
@@ -59,12 +58,41 @@ def contains(self, value):
         curr_node = curr_node.next_node
 
   def get_max(self):
-    pass
     # set the currrent highest to the first
+    current_highest = self.head
+    # set current node to the first node
+    current_node = self.head
     # while True to create a loop
-    # check if the next node is higher and if so
-      # set current_highest to current node
-      # set current node to next node
-    #else
-      # set current node to next node
-    # if next node = None return current_highest
+    while True:
```

Comment:
Great logic implementing a while loop for your get max, just make sure to keep your indentations in place to keep your code clean